src/python/antares_xpansion/general_data_processor.py

```diff
@@ -78,13 +78,13 @@ def backport_playlist(self, ini_file_backup, writer, playlist_options: dict):
         get_general_data_ini_file, set_general_data_ini_file
     )
 
-    def _get_values_to_change_general_data_file(self):
+    def _get_values_to_change_general_data_file(self, memory_mode=False):
         optimization = "optimization"
         general_section = "general"
         output_section = "output"
 
         return {
-            (optimization, "include-exportmps"): "optim-1",
+            (optimization, "include-exportmps"): "false" if memory_mode else "optim-1",
             (optimization, "include-exportstructure"): "true",
```

Comment:
In the future, should we also pass the information contained in `ts-numbers` in memory ?

Reply:
Ideally yes.